pkg/iac/terraform/state/enumerator/azurerm.go

```diff
@@ -82,6 +82,10 @@ func (s *AzureRMEnumerator) Enumerate() ([]string, error) {
 	}
 
 	if err := pager.Err(); err != nil {
+		storageErr, ok := err.(azblob.ResponseError)
+		if ok && storageErr.RawResponse() != nil {
```

Comment:
Could be a one line `if storageErr, ok := err.(azblob.ResponseError); ok && storageErr.RawResponse() != nil {`